docs/gettingstarted/index.rst

```diff
@@ -173,7 +173,9 @@ Getting started with Flocker
             you@laptop:~$ git clone \
               https://github.com/clusterhq/vagrant-flocker && \
               cd vagrant-flocker && \
-              vagrant up
+              vagrant up && \
+              [ -e "${SSH_AUTH_SOCK}" ] || eval $(ssh-agent) && \
+              ssh-add ~/.vagrant.d/insecure_private_key
```

Comment:
What we found yesterday is that if an agent isn't running, it will be started in the second console that was used for starting the vagrant nodes, but the agent socket path and the loaded key will not then be available in the first console where the user has installed `flocker-deploy`.

Those of us testing on Ubuntu / Fedora usually don't see this because the desktop an alternative key agent eg gnome-keyring-daemon, which exports the SSH_AUTH_SOCK environment variable, which is  inherited by all child shells.

```
[~/flocker-tutorial-ubuntu]$ sudo lsof -p $(pidof gnome-keyring-daemon) | grep "$SSH_AUTH_SOCK" 
gnome-key 1696 richard   14u     unix 0xffff8800c5ffcd00      0t0   30296 /run/user/1000/keyring-nkE1ut/ssh
```